fido2/server.py

```diff
@@ -49,13 +49,33 @@ class ATTESTATION(six.text_type, Enum):
     DIRECT = 'direct'
 
 
+@unique
+class USER_VERIFICATION(six.text_type, Enum):
+    DISCOURAGED = 'discouraged'
+    PREFERRED = 'preferred'
+    REQUIRED = 'required'
+
+
 class Fido2Server(object):
-    def __init__(self, rp, attestation=None, verify_origin=None):
+    def __init__(
+            self,
+            rp,
+            attestation=None,
+            verify_origin=None,
+            resident_key=False,
+            user_verification=USER_VERIFICATION.PREFERRED
+    ):
+        if user_verification not in USER_VERIFICATION:
```

Comment:
I'd change this to make it possible to pass in a string as well. Now, for example:
`'discouraged' in USER_VERIFICATION == False`.

Maybe change it to: `self.user_verification = USER_VERIFICATION(user_verification)`?
That has the benefit of already throwing a ValueError if something weird is passed.

Reply:
I agree, no need to import the enum, which is nice.